doc/user_guide/display.rst

```diff
@@ -0,0 +1,307 @@
+.. _displaying-charts:
+
+Displaying Charts
+=================
+
+Altair provides a high-level API for creating visualizations that are
+encoded as declarative JSON objects. To display a visualization, those JSON
+objects must be diplayed using the `Vega-Lite`_ and `Vega`_ JavaScript packages.
+This step of displaying a Vega-Lite or Vega JSON object is encoding in renderer
```

Comment:
I’m not sure how to parse this sentence.

Reply:
better?